pkg/action/build-uki.go

```diff
@@ -182,8 +182,14 @@ func (b *BuildUKIAction) createSystemdConf(sourceDir string) error {
 		// This is the one name that has nothing added, just the version
 		finalEfiConf = nameFromCmdline(constants.UkiCmdline+" "+constants.UkiCmdlineInstall) + ".conf"
 	}
+
+	secureBootEnroll := viper.GetString("secure-boot-enroll")
+	if secureBootEnroll == "" {
```

Comment:
no need to do this if the flags are binded no? I thought we fixed this?

Reply:
You are right. Removing.